cluster-support-bot.py

```diff
@@ -7,6 +7,20 @@
 import slack
 import telemetry
 
+import prometheus_client as prom
+import re
```

Comment:
nit: from [PEP 8][1]:

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

So this `re` should go just after `os`, and `import prometheus_client` should go just after `hydra`.  I'd also rather drop `as prom`, because if anyone is already familiar with `prometheus_client`, they won't need to perform the `prom` -> `prometheus_client` translation when reading later code consuming the package.

[1]: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
would prometheus_client / slack fall in bucket 2 and hydra / telemetry in bucket 3?

Reply:
> would prometheus_client / slack fall in bucket 2

Yup.

> and hydra / telemetry in bucket 3?

I'm fine either way, so your current commit is fine.